pages/news/2020-08-25-new-in-2-3-hotcue-colors.html

```diff
@@ -0,0 +1,141 @@
+
+Personal preferences differ though, and it's probably impossible to suit everyones taste.
+This is why - unlike most other DJ software - you are not restricted to the single predefined color palette we picked.
+
+If you previously used Serato DJ Pro, Rekordbox, Traktor or VirtualDJ and like their color palettes better ours, we've got you covered.
+You can also use one of their palettes instead.
+
+![Hotcue color palette selection]({% static "/images/news/hotcue-colors-palette-combobox.png" %})
+
+But it doesn't stop there:
+We also added a powerful color palette editor that can be used to rearrange and customize these palettes and even design *your own palette* using any RGB colors you like.
+
+![Hotcue color palette editor]({% static "/images/news/hotcue-colors-palette-editor.png" %})
+
+## Automatic Coloring
+
+When setting a hotcue, a single default color assigned that you can select in the preferences.
+You can then go ahead and change the color to your liking.
+
+While assigning a custom color to cue points after you set them is a nice feature, coloring all your cue points this way can be tedious and a lot of extra work.
+If you want, Mixxx 2.3 can color new hotcues automatically for you - based on the hotcue number.
+
+A small example:
+Say you want to set a cue point before the main vocal part, and you want it colored "blue".
+You can configure Mixxx to always assign "blue" to the hotcues number 5, and then just set hotcue number 5 at the desired position - done.
+
+If you do this, that also means that a hotcue of the same type (e.g. "before main vocals") does not only have the same color but is also assigned to the same hotcue pad.
+Hence, the possibility of getting confused during a gig is reduced even more than when just using colors as visual aid.
+
+## Bulk replace old hotcue colors
+
+At some point, you might feel like your current scheme for color code hotcues lacking and unintuitive.
+Maybe you simply don't like the colors anymore and would like to switch to a different palette.
+Or you realize that you find the colorful hotcue buttons actually distracting and want to get rid of them.
+If that happens, you may want to bulk replace the colors of existing hotcue in your library.
+
+We're not aware that any of our competitors in the DJ software market have such an option - Mixxx 2.3 has.
+You can find a dialog for replacing colors on the "Colors" page in the preferences window.
+
+It allows bulk replacing hotcue colors for hotcues in your library and can be used to
+- replace colors by hotcue number ("Set the color of all hotcues with number 5 to blue"),
+- replace colors by current color ("Set the color of all red hotcues to blue"),
+- combine the two conditions ("Set the color of all red hotcues with index 5 to blue"),
+- negate them ("Set the color of all hotcues that are red but don't have number 5 to blue")  or
+- just replace all hotcue colors in your library unconditionally ("Set the color of all hotcues to blue").
```

Comment:
on https://deploy-preview-83--mixxx-website.netlify.app/news/2020-08-25-new-in-2-3-hotcue-colors/ this is not as a list but on a continious line.